Protocol.md

```diff
@@ -91,7 +91,7 @@ The following are features of Data and Metadata.
 The RSocket protocol uses a lower level transport protocol to carry RSocket frames. A transport protocol MUST provide the following:
 
 1. Unicast [Reliable Delivery](https://en.wikipedia.org/wiki/Reliability_(computer_networking)).
-1. [Connection-Oriented](https://en.wikipedia.org/wiki/Connection-oriented_communication) and preservation of frame ordering. Frame A sent before Frame B MUST arrive in source order. i.e. if Frame A is sent by the same source as Frame B, then Frame A will always arrive before Frame B. No assumptions about ordering across sources is assumed.
+1. Stream-Oriented (similar to [connection-oriented](https://en.wikipedia.org/wiki/Connection-oriented_communication)) preservation of frame ordering for the same stream within a multiplexed connection. If frame A is sent before frame B, within the same stream, on the same connection, then frame A MUST arrive before frame B. No assumptions are made about ordering across different streams within the same multiplexed connection.
```

Comment:
I don't think we should drop connection-oriented completely here. Isn't a TCP socket still an option but with known limitations like head of line blocking.

Reply:
I guess the point here is to emphasize that we don't need connection-oriented message sending anymore but we rather need stream-oriented. If connection support stream-oriented message sending via connection-oriented message sending - it is fine

Reply:
Maybe worth a clarifying note.  What does this spec mean for something without streams?

Reply:
yeah. Make sense